### ISSUE TITLE
Mod crashes game upon launch

### ISSUE THREAD

**Issue by veeonthewall** [Deftu/Screencapper#4]
<img width="566" alt="Screen Shot 2022-07-20 at 7 48 44 PM" src="https://user-images.githubusercontent.com/105337781/180106378-5dcf0d48-e71d-4c7c-b876-ab4f21d884c2.png">
This is the error I am given.

**Comment by ifritdiezel** [comment]
i'm likely having the same issue, here's a log
```
Caused by: java.lang.NoClassDefFoundError: gg/essential/universal/ChatColor
	at xyz.deftu.screencapper.config.ScreencapperConfig.<init>(ScreencapperConfig.kt:20)
	at xyz.deftu.screencapper.config.ScreencapperConfig.<clinit>(ScreencapperConfig.kt)
	at xyz.deftu.screencapper.Screencapper.onInitializeClient(Screencapper.kt:22)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 15 more
Caused by: java.lang.ClassNotFoundException: gg.essential.universal.ChatColor
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 19 more
```

**Comment by Deftu** [comment]
I'm so sorry for seeing this so late! I wasn't notified about it via email as GitHub usually should do.

This is an odd issue. I'll look into it for 1.1.0.

**Comment by Deftu** [comment]
This has now been fixed in the release of 1.1.0

**Comment by veeonthewall** [comment]
Thanks a bunch!
